### ISSUE TITLE
encoding problem with feed body with postgresql 

### ISSUE THREAD

**Issue by UnnamedHero** [owncloud/news#801]
I have issues with feed body encoding. Look at this screenshot, for example.

![owbcloud_news_fail](https://cloud.githubusercontent.com/assets/1024934/7809322/cc7dc0c8-0388-11e5-9240-8678f064c93a.png)

Every other feed have same issue.

My configuration is run on  centos 7.1, php-fpm, nginx, postgresql. Owncloud 8.0.3, News 5.2.8.
.user.ini in owncloud dir contains:
```
mbstring.internal_encoding=UTF-8
mbstring.language=utf-8
mbstring.http_input=utf-8
mbstring.http_output=utf-8
```
nginx has this in site configuration:
```
charset utf-8;
```
postgresql has proper encoding
```
   Name    |  Owner   | Encoding |   Collate   |    Ctype    |   Access privileges   

 qqodb     | qqo      | UTF8     | ru_RU.UTF-8 | ru_RU.UTF-8 | =Tc/qqo              +
           |          |          |             |             | qqo=CTc/qqo
```

system locale 
```
LANG=ru_RU.UTF-8
LC_CTYPE="ru_RU.UTF-8"
LC_NUMERIC="ru_RU.UTF-8"
LC_TIME="ru_RU.UTF-8"
LC_COLLATE="ru_RU.UTF-8"
LC_MONETARY="ru_RU.UTF-8"
LC_MESSAGES="ru_RU.UTF-8"
LC_PAPER="ru_RU.UTF-8"
LC_NAME="ru_RU.UTF-8"
LC_ADDRESS="ru_RU.UTF-8"
LC_TELEPHONE="ru_RU.UTF-8"
LC_MEASUREMENT="ru_RU.UTF-8"
LC_IDENTIFICATION="ru_RU.UTF-8"
LC_ALL=
```
What is wrong?

**Comment by Raydiation** [comment]
Feed url?

**Comment by UnnamedHero** [comment]
http://www.anekdot.ru/rss/export20.xml
http://www.opennet.ru/opennews/opennews_all.rss

**Comment by Raydiation** [comment]
Ok, got it working fine on my installation and postgresql (5.3.4, newest ownCloud from git)

Could you try to switch to en_US.UTF-8 since that should make no difference?

**Comment by Raydiation** [comment]
@cosenal @icewind1991 @PVince81 @LukasReschke can you reproduce this?

**Comment by Raydiation** [comment]
Probably broken unicode setup on your postgres install.

**Comment by UnnamedHero** [comment]
I've tried with another locale - en_US.UTF-8. Same thing
I've tried reinitialize whole DB with 
```
initdb -D "my_path" --locale=en_US.UTF-8 
```
or with ru_RU.UTF-8
I've got same thing - bad encoding in feed body, while feed header is ok. And I don't think my postgres installation is broken.

I'll be waiting for a official OC 8.1 release, and I'll try News >=5.3.4.

**Comment by icewind1991** [comment]
I never had issues with japanese characters in the feed body.

all locales on en_US.UTF-8

**Comment by UnnamedHero** [comment]
confirm fixed with News 5.3.5, and OC 8.1_beta

**Comment by Raydiation** [comment]
\o/
